.github/workflows/interface-unit-tests.yml

```diff
@@ -11,7 +11,7 @@ on:
         required: true
         type: string
       pipeline_mode:
-          description: The pipeline mode can be unit-tests, benchmarks, or reference-benchmarks
+          description: The pipeline mode can be unit-tests, update-durations, benchmarks, or reference-benchmarks
```

Comment:
This new pipeline mode is used to selectively run jobs that are needed for updating the durations files.